docs/topics/Gateway.md

```diff
@@ -790,30 +790,30 @@ A user's presence is their current state on a guild. This event is sent when a u
 >info
 >The streaming type currently only supports Twitch. Only `https://twitch.tv/` urls will work.
 
-###### Game Timestamps
+###### Activity Timestamps
 
 | Field | Type | Description |
 |-------|------|-------------|
-| start? | int | Epoch millis of start |
-| end? | int | Epoch millis of end |
+| start? | int | unix time (in milliseconds) of when the activity started |
+| end? | int | unix time (in milliseconds) of when the activity ends |
 
-###### Game Party
+###### Activity Party
 
 | Field | Type | Description |
 |-------|------|-------------|
-| id? | string | The ID, relevant for joining the party |
-| size? | array of two integers | First element is the current size, second the max `[2, 5]` |
+| id? | string | the ID of the party |
+| size? | array of two integers (current\_size, max\_size) | used to show the party's current and maximum size |
 
-###### Game Assets
+###### Activity Assets
 
 | Field | Type | Description |
 |-------|------|-------------|
-| large_image? | snowflake | ID for an asset of the application |
-| large_text? | string | Text displayed when hovering over the large image of the presence |
-| small_image? | snowflake | ID for an asset of the application |
-| small_text? | string | Text displayed when hovering over the small image of the presence |
+| large_image? | snowflake/string | the ID for an asset of the application |
```

Comment:
you should just say it's a string and that it's a large image asset of the activity

Reply:
I asked 6 days ago whether it should be just `string`